tests/js/spec/components/search/sources/routeSource.spec.jsx

```diff
@@ -9,7 +10,13 @@ describe('RouteSource', function() {
 
   it('can find a route', async function() {
     const mock = jest.fn().mockReturnValue(null);
-    wrapper = mount(<RouteSource query="password">{mock}</RouteSource>);
+
+    const {organization, project} = initializeOrg();
+    wrapper = mount(
+      <RouteSource query="password" {...{organization, project}}>
```

Comment:
Does this require org and project now?

Reply:
Not strictly, but for the tests it does otherwise it now won't return results (since the results would be wrong without org / project)

Reply:
The tests passed before, but the links would be wrong since they would be missing org / project slug